pkg/ui/src/views/reports/containers/debug/index.tsx

```diff
@@ -96,8 +100,9 @@ export default function Debug() {
           </p>
         </InfoBox>
 
-        <div className="debug-header__license-type">
+        <div className="debug-header__annotations">
           <LicenseType />
+          <DebugAnnotation label="Web server" value={ `n${NODE_ID}` } />
```

Comment:
would prefer using "Gateway node", but that's a bikeshed…

Reply:
Initially that's what I had.  Then I edited it to "Gateway node (initial load)" to clarify that it's the node that the first request hit.  Then I thought maybe "Web server"/"API server" was a more concise way to make that distinction.  I don't really know.